append_deps.py

```diff
@@ -0,0 +1,29 @@
+import zipfile
+import os
+import shutil
+import json
+import tempfile
+
+directories = ["artifacts/release/", "artifacts/snapshot/"]
+
+BASE_DIR = dir_path = os.path.dirname(os.path.realpath(__file__))
+
+for dir in directories:
+    dir = os.path.join(BASE_DIR, dir)
+    os.chdir(dir)
+    for f in [a for a in os.listdir('./')]:
+        name, ext = os.path.splitext(f)
+        if ext == '.zip' and 'withdeps' not in name:
+            with tempfile.TemporaryDirectory() as tmpdirname:
+                print(f"Appending deps for {name}, working dir {tmpdirname}")
+                os.chdir(tmpdirname)
```

Comment:
Maybe just for safety ```os.chdir(os.path.abspath(tmpdirname))```. It *should* be abspathed here, but this is something I'm always worried about (personally).